bin/inference/pycbc_inference_plot_geweke

```diff
@@ -93,7 +92,6 @@ fig = plt.figure()
 # loop over each parameter
 for param, label in zip(params, labels):
     logging.info("Plotting parameter %s", param)
-
     # loop over walkers
     for j in walkers:
         logging.info("Plotting walker %d of %d", j, nwalkers)
```

Comment:
At line 100, below, do:
```
y = fp.samples_from_cli(opts, parameters=param, walkers=j)
```
instead of
```
         y = fp.read_samples(param, walkers=j,
                             thin_start=opts.thin_start,
                             thin_interval=opts.thin_interval,
                             thin_end=opts.thin_end)
```